tests/phpunit-env1.xml

```diff
@@ -1,10 +1,40 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.github.com/sebastianbergmann/phpunit/master/phpunit.xsd"
-	convertErrorsToExceptions="true"
-	convertNoticesToExceptions="true"
-	convertWarningsToExceptions="true"
-	bootstrap="phpunit-bootstrap.php">
-	<php>
-		<server name="PKP_MOCK_ENV" value="env1"/>
-	</php>
+<phpunit
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    cacheResult="true"
+    beStrictAboutOutputDuringTests="true"
+    backupGlobals="false"
+    backupStaticAttributes="false"
+    bootstrap="phpunit-bootstrap.php"
+    colors="true"
+    convertErrorsToExceptions="true"
+    convertNoticesToExceptions="true"
+    convertWarningsToExceptions="true"
+    processIsolation="false"
+    stopOnFailure="true"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
+    <testsuites>
+        <testsuite name="Common Test Suite">
+            <directory suffix="Test.php">classes</directory>
+        </testsuite>
+    </testsuites>
+    <coverage>
```

Comment:
Does the code coverage stuff work? We used to have a test suite for this using Openstack but it's long since removed.

Reply:
Yes. I ran the following command on OMP and generated the following report.

```
export XDEBUG_MODE=coverage && php lib/pkp/lib/vendor/phpunit/phpunit/phpunit --configuration lib/pkp/tests/phpunit-env1.xml  --filter testRoute --coverage-html "lib/pkp/tests/results/coverage-html"
```
![Screenshot from 2021-12-13 22-47-06](https://user-images.githubusercontent.com/1243634/145917984-a55d14f1-fb68-45d1-82fc-19eb979d0180.png)